### DISCUSSION TITLE
ESP32 DevKit C V4 with ESP Wrover module pin assignment / NRF module IRQ line no connection

### DISCUSSION THREAD

**Discussion by flying-tux** [tbnobody/OpenDTU#658] [Q&A]
When using ESP32 DevKit C V4 with an ESP32-WROVER module on it, please be aware that GPIO16 is unavailable.

[Espressif ESP32 DevKit C V4 get started guide](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/hw-reference/esp32/get-started-devkitc.html)

This leads to an apparent correctly working nrf24l01+ radio module in web interface but missing data from hoymiles device.

I moved the pin assignment for the IRQ line to GPIO15 which should be available on all similar boards. Maybe this could be the standard or example configuration too?

Best regards 

Christian

**Comment by tbnobody** [comment]
Do you already have a working json file? Then I would add it to the repository (you could also send me a PR with the json file)

**Comment by flying-tux** [comment]
I just replaced the nrf irq section from 16 to 15 and it is working without any issues, please find attached. I didn't open a pull request at this point as it would break the wiring diagram and documentation without changing them accordingly.

[pin_mapping_wrover.zip](https://github.com/tbnobody/OpenDTU/files/10892870/pin_mapping_wrover.zip)
